cloudstack/resource_cloudstack_network_offering.go

```diff
@@ -125,34 +125,34 @@ func resourceCloudStackNetworkOfferingUpdate(d *schema.ResourceData, meta interf
 
 	}
 
-	// Check if the guest ip type is changed and if so, update the virtual machine
-	if d.HasChange("guest_ip_type") {
-		log.Printf("[DEBUG] Guest ip type changed for %s, starting update", name)
+	// Check if maxconnections is changed and if so, update the network offering
```

Comment:
These fields are not defined in the schema. Let's merge this PR with https://github.com/apache/cloudstack-terraform-provider/pull/205/?